README.md

```diff
@@ -116,31 +121,37 @@ This will create a PUT named "test", and a TPP named "partner" which can be used
 example `test-configuration.yaml` in [`test-network-function`](https://github.com/test-network-function/test-network-function)
 will use this local infrastructure by default.
 
-To verify `partner` and `test` pods are running: 
+To verify `partner` and `test`  pods are running: 
 
 ```shell-script
 oc get pods -n tnf -o wide
 ```
 
-You should see something like this (note that the 2 test pods are running on different nodes due to a anti-affinity rule):
+You should see something like this in case of running it on minikube (note that the 2 test pods are running on different nodes due to a anti-affinity rule):
 ```shell-script
 NAME                       READY   STATUS    RESTARTS   AGE    IP           NODE           NOMINATED NODE   READINESS GATES
 partner-68cf756959-tp2c5   1/1     Running   0          110s   10.244.1.7   minikube-m02   <none>           <none>
 test-7799cc9677-6d8qz      1/1     Running   0          110s   10.244.1.8   minikube-m02   <none>           <none>
 test-7799cc9677-rv8nv      1/1     Running   0          110s   10.244.0.5   minikube       <none>           <none>
 ```
+And to see this in case of oc:
+```shell-script
+NAME                       READY   STATUS    RESTARTS   AGE
+node-master                1/1     Running   0          8d
```

Comment:
in the PR, we actually removed the node-master. It's no longer needed. Please check last commit

Reply:
i deleted it